packages/contracts/docs/platform.md

```diff
@@ -0,0 +1,189 @@
+# Smart Contracts Documentation
+
+## Overview
+
+This documentation provides details on the smart contracts used in the system, including their properties and methods. It also includes integration notes for connecting these contracts with other components like `maci-subgraph`.
+
+## Contracts
+
+### Contract: `EASRegistryManager`
+
+The `EASRegistryManager` contract handles requests for registry management. Users send requests to this contract, and a coordinator approves or rejects these requests.
+
+#### Properties
+
+- **contractId**: `uint256` - Unique identifier for the contract.
+- **owner**: `address` - The address of the contract owner.
+
+#### Methods
+
+- **getOwner()**: `address` - Returns the address of the owner.
+- **setOwner(newOwner: address): void** - Sets a new owner for the contract.
+
+### Contract: `EASRegistry`
+
+The `EASRegistry` contract allows public access to information about projects. It is designed for reading project data, but if a subgraph is integrated, this contract may not be necessary for frontend use.
+
+#### Properties
+
+- **easId**: `uint256` - Unique identifier for the EAS.
```

Comment:
is this a certain EAS schema id? maybe it's better that we provide example of how should it be like here?